src/services/Safeplace.ts

```diff
@@ -2,7 +2,7 @@ import IUser from '../components/interfaces/IUser';
 import { createHttpConfig } from '../http-common';
 
 class Safeplace {
-    private url: string = "http://api.safely-app.fr/safeplace";
+    private url: string = "https://api.safely-app.fr/safeplace";
```

Comment:
Just a suggestion but wouldn't be better to setup some sort of environment file where you would have your api url store in it ?

It's not really needed, it will be just more convenient

Reply:
Now that we have the same url everywhere this seems a good idea.

Reply:
With the commit fcb4e923aecc289aaa8199865e0da87699d7c847, It's now working fine with a environment file!